File: fix.py
import base64, os
home = os.environ['HOME']
with open(home+'/baxto-pro/gensite.js') as f:
    content = f.read()
start = content.index("'") + 1
end = content.rindex("'")
b64 = content[start:end]
html = base64.b64decode(b64)
with open(home+'/baxto-pro/public/index.html','wb') as f:
    f.write(html)
print('Listo!', len(html), 'bytes')

File: fix.js
const fs = require('fs');
const d = fs.readFileSync(process.env.HOME+'/baxto-pro/gensite.js','utf8');
const b = d.split("'")[1];
fs.writeFileSync(process.env.HOME+'/baxto-pro/public/index.html', Buffer.from(b,'base64'));
console.log('Listo! ' + b.length + ' chars');
